docs/platform_operators/external-blobstore.md

```diff
@@ -0,0 +1,34 @@
+# Using external blobstore
+
+1. Create the necessary buckets using `aws-cli`:
+
+    ```
+    REGION=<your-region>
+    aws s3api create-bucket --bucket <bucket-name-buildpack> --region "$REGION" --create-bucket-configuration LocationConstraint="$REGION" --acl private
+    aws s3api create-bucket --bucket <bucket-name-droplet>   --region "$REGION" --create-bucket-configuration LocationConstraint="$REGION" --acl private
+    aws s3api create-bucket --bucket <bucket-name-package>   --region "$REGION" --create-bucket-configuration LocationConstraint="$REGION" --acl private
+    aws s3api create-bucket --bucket <bucket-name-resource>  --region "$REGION" --create-bucket-configuration LocationConstraint="$REGION" --acl private
+    ```
+
+    *Note*: The name of your buckets (`package_directory_key`, `droplet_directory_key`, `resource_directory_key`, `buildpack_directory_key`) need to be unique per region! Also check your needed/supported `aws_signature_version` and configure it accordingly.
```

Comment:
We think that the names of the buckets have to be globally unique, not just per region.